arangod/Agency/Agent.cpp

```diff
@@ -327,6 +327,15 @@ index_t Agent::confirmed(std::string const& agentId) const {
 void Agent::reportIn(std::string const& peerId, index_t index, size_t toLog) {
   auto startTime = steady_clock::now();
 
+  if (index == 0) {
```

Comment:
Let's add a comment here to say that this happens if and only if we are receiving a response for an empty appendentriesRPC.